### PR TITLE
MULE-11603 Add DW performance tests. Add single very limited test to …

### PR DESCRIPTION
**Pull request by dfeist**
…get initial number.

### CODE REVIEW

File: tests/performance/pom.xml [hunk 1]
```diff
@@ -111,6 +111,12 @@
             <classifier>mule-service</classifier>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mule.services</groupId>
+            <artifactId>mule-service-weave</artifactId>
+            <classifier>mule-service</classifier>
+            <version>${project.version}</version>
+        </dependency>
```
Comment (pablokraan) [inline tests/performance/pom.xml]: provided?

File: tests/performance/src/main/java/org/mule/el/SimpleExpressionBenchmark.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  @Param({"mel:payload", "payload"})
+  private String expression;
+
+  private MuleContext muleContext;
+  private Event event;
+
+  @Setup
+  public void setup() throws MuleException {
+    muleContext = createMuleContextWithServices();
+    event = Event.builder(DefaultEventContext.create(createFlow(muleContext), ""))
```
Comment (pablokraan) [inline tests/performance/src/main/java/org/mule/el/SimpleExpressionBenchmark.java]: Use static import

File: tests/performance/src/test/java/org/mule/test/core/el/ExpressionBenchmarkAssertionTestCase.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  public void evaluatePayloadMEL() {
+    runAndAssertBenchmark(SimpleExpressionBenchmark.class, "evaluatePayload", 1, singletonMap(EXPRESSION_PARAM,
+                                                                                              new String[] {
+                                                                                                  "mel:payload"}),
+                          700, NANOSECONDS, 2000);
+  }
+
+  @Test
+  public void evaluatePayloadDW() {
+    // DW numbers are significantly below MEL currently but by adding assertions we avoid any regressions
```
Comment (pablokraan) [inline tests/performance/src/test/java/org/mule/test/core/el/ExpressionBenchmarkAssertionTestCase.java]: There should be a reference to the jira tracking this problem here.

Comment (dfeist) [inline tests/performance/src/test/java/org/mule/test/core/el/ExpressionBenchmarkAssertionTestCase.java]: Yep, I'll check with @afelisatti  to see if we have one..

File: tests/performance/pom.xml (hunk 1 above)
Comment (dfeist) [inline tests/performance/pom.xml]: If i put it provided then it's not included in the benchmark uber jar.

**Pull request merged**